macros/test-helpers/check_seed_macro_legacy.sql

```diff
@@ -54,8 +54,8 @@
     matching_count_test as (
         select
             'matched records count' as test_description,
-            count(model_{{seed_matching_columns[0]}}) as `result_model`,
-            1 as `expected_seed`,
+            count(model_{{seed_matching_columns[0]}}) as result_model,
+            1 as expected_seed,
```

Comment:
I think remove the ` in check_seed_macro_legacy to fix errors in dbt-test.

Reply:
interesting, this was failing on the spark CI step? how did it run previously then?

Reply:
The failing error looks like this:
https://github.com/duneanalytics/spellbook/actions/runs/5574212027/jobs/10182523270#step:11:65

Reply:
that failure is happening on the dunesql CI step, but that macro is marked as `legacy`. it should be running on spark instead. you likely need to update your schema files to call the non-legacy seed macro, so it runs the correct query

Reply:
please follow example in #3804 for how to use seeds in migration. we should ultimately be able to remove changes to this legacy file from this PR.

Reply:
I will recover this file.